modules/social_features/social_group/modules/social_group_default_route/social_group_default_route.install

```diff
@@ -16,3 +18,18 @@ function social_group_default_route_install() {
   // Set the weight of this module.
   module_set_weight('social_group_default_route', $this_module);
 }
+
+/**
+ * Add "Enable tab settings" field to group entity.
```

Comment:
@rochek03 I don't think the requirement of such a field in the original ticket.
Can you explain why you added it and if we can avoid adding new fields?

Reply:
@ribel fixed